members/forms.py

```diff
@@ -1,19 +1,36 @@
 from django import forms
+from django.utils import timezone
 from django.contrib.auth.forms import UserCreationForm
 from django.contrib.auth.models import User
+from iquise.utils import random_password
 
-from members.models import Person
+def this_year():
+    return timezone.now().year
 
+class JoinForm(forms.Form):
+    LEVELS = ("Highschool", "Undergraduate", "Graduate", "PostDoc", "Professor", "Professional", "Retired")
```

Comment:
"unemployed" rip

Reply:
we shouldn't add that but we do get a nontrivial number of people who are not any of these currently trying to sign up

Reply:
Agreed it is worth considering this further. Could also have a checkbox asking if MIT alum